0_RootFS/llvm_common.jl

```diff
@@ -204,7 +206,7 @@ function llvm_dependencies(; kwargs...)
     return [
         Dependency("Zlib_jll"),
         Dependency("XML2_jll"),
```

Comment:
Uhm, I think we should fix the xml2 version here, maybe depending on the llvm version for backward compatibility: something like xml2 ~2.13 for llvm <=18, and ~2.14 for llvm >= 19